CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Change log
+
+## [Unreleased]
+
+### New features
+
+* Add `Gocov` support
```

Comment:
I like that, and remove the `### New features`/`### Bug fixes` header since they're won't be needed anymore.